src/viur/core/modules/user.py

```diff
@@ -450,7 +441,7 @@ def pwrecover(self, recovery_key: str | None = None, skey: str | None = None, *a
                 )
             )
 
-        if user_skel["status"] != Status.ACTIVE:  # The account is locked or not yet validated. Abort the process.
+        if user_skel["status"] < Status.ACTIVE:  # The account is locked or not yet validated. Abort the process.
```

Comment:
We have never specified anywhere which value ranges mean what. As far as I'm OK with that, it can be checked that way. But then there should be an explanation somewhere.